### DISCUSSION TITLE
Dividend and Interest performance calculation

### DISCUSSION THREAD

**Discussion by fib1985** [ghostfolio/ghostfolio#3966] [Q&A]
Hello everyone,

a question regarding dividends and interest: are they taken into account when calculating portfolio performance? Or are they excluded from this calculation? thank you.

**Comment by dtslvr** [comment]
Hi @fib1985

Dividend and interest aren’t included in the current performance calculation, but work is in progress (https://github.com/ghostfolio/ghostfolio/pull/3857).

**Reply by fib1985** [reply]
Great news! Ghostfolio is developing very quickly!
